apps/beeswax/src/beeswax/server/dbms.py

```diff
@@ -806,7 +807,10 @@ def drop_table(self, database, table):
 
   def load_data(self, database, table, form_data, design, generate_ddl_only=False):
     hql = "LOAD DATA INPATH"
-    hql += " '%(path)s'" % form_data
+    source_path = "%(path)s" % form_data
+    if source_path.lower().startswith("abfs"): #this is to check if its using an ABFS path
```

Comment:
Good to know! I don't see a cleaner way for now (until we refactor the importer/file browser).